pyproject.toml

```diff
@@ -53,11 +53,11 @@ version-file = "src/execnet/_version.py"
 
 [tool.hatch.build.targets.sdist]
```

Comment:
Couldn't find an option "include everything that is version controlled" as we had with `setuptools-scm`. 😞